operate/common/src/main/java/io/camunda/operate/zeebe/PartitionSupplier.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.zeebe;
+
+import io.camunda.operate.util.Either;
+
+public interface PartitionSupplier {
```

Comment:
We need to have a shared module (something like `common-webapps`) to put the shared code between the webapp modules